core-bundle/src/Picker/AbstractPickerProvider.php

```diff
@@ -131,6 +131,15 @@ protected function getInsertTag(PickerConfig $config, string $default): string
         return $default;
     }
 
+    /**
+     * Provides a shortcut get the "insertTag" extra value for child classes and
+     * split them at the placeholder (%s).
+     */
+    protected function getInsertTagChunks(PickerConfig $config, string $default): array
```

Comment:
Can we not set a default value for the second argument? `string $default = static::DEFAULT_INSERTTAG`

Reply:
And what if I forget to set this in my child? I would only notice it at runtime which makes the whole idea of an `abstract` class and interfaces pointless 😄

Reply:
It would use the constant of the parent class then.

Reply:
Which does not exist because what value would it provide? 😄

Reply:
Ok, then why did you make the constant protected instead of private? Also, the naming is misleading: `DEFAULT_INSERTTAG` implies that there are constants for other, non-default insert tags. We should change it to just `INSERTTAG`.

Reply:
So that they can be replaced by a class extending a specific picker? I don't think the name is misleading, it's the default insert tag if no other configuration was provided.